projects/transformers/README.md

```diff
@@ -35,6 +35,9 @@ Known issues, to be investigated/fixed:
 
 > Finetuning is important to evaluate a language model effectiveness when being used for transfer learning in downstream tasks. However, it is not a reliable metric to use for optimization. These numbers will vary with a significant margin of error between two different runs. Use perplexity instead to guide hyperparameter search and development of language models.
 
+Known Bugs
```

Comment:
I haven't observed this before, and I ran dozens of finetuning runs,so seems like a bug introduced by a recent version. Is there a way to set wandb logging default to False in finetuning, so we avoid this? Most of the finetuning runs we only care about the end result.

Also, do you believe this will persist once we introduce deepspeed?

Reply:
I believe one sets `WANDB_DISABLED=false`. Are you suggesting it to be set automatically? I'm curious if the problem will persist. It happened to me only one day, so maybe I had did something wrong. I think if you and Ben encounter it, then maybe we should disable wandb automatically.

Reply:
I'm not sure how it will interact with DeepSpeed.